src/hooks/useSubaccount.tsx

```diff
@@ -436,6 +436,15 @@ export const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: Lo
           onSuccess?.(data);
         } else {
           onError?.(data, { errorStringKey: parsingError?.stringKey });
+
+          const { placeOrderPayloads } = data || {};
+          placeOrderPayloads &&
+            placeOrderPayloads.toString() != '[]' &&
+            placeOrderPayloads.toArray().forEach((payload: HumanReadablePlaceOrderPayload) => {
+              if (payload.clientId !== undefined) {
+                dispatch(removeUncommittedOrderClientId(payload.clientId));
+              }
+            });
```

Comment:
will it be any clearer if we have another function that takes in an array of clientIds instead?

so we get the failed order clientIds first -> filter out those clientIds?

Reply:
hmm do you mean having a `submittedOrdersFailed` in addition to `submittedOrderFailed`? not sure how strongly I feel about this, because at least for SL/TP, I think we should only ever have one payload returned per error 🤔 (it's just a list because the `HumanReadableTriggerOrderPayloads` is a shared type in abacus - but we send out a new message for each payload).